tests/Parsing/ParsingTest.elm

```diff
@@ -0,0 +1,98 @@
+module Parsing.ParsingTest exposing (parseTest)
+
+import Elm.Parser
+import Elm.Processing
+import Elm.Syntax.File
+import Elm.Syntax.Range
+import Expect
+import Intermediate
+import Models
+import Test
+import TestUtil
+
+
+parseTest : Test.Test
+parseTest =
+    let
+        checkExpectation expected modString =
+            case TestUtil.stringToIntermediate modString of
+                Just parsed ->
+                    Expect.equal parsed expected
+
+                Nothing ->
+                    Expect.fail "Failed to parse the module!"
+    in
+    Test.describe "Test the parsing function in its entirety."
```

Comment:
Please be more specific here, "in its entirety" is non-informative.

For example, ask yourself what the opposite would mean: how would you describe a test that does _not_ test the parsing function in its entirety? That answer will then give you a hint how to better describe this test.